content/en/docs/concepts/containers/images.md

```diff
@@ -157,6 +157,58 @@ that Kubernetes will keep trying to pull the image, with an increasing back-off
 Kubernetes raises the delay between each attempt until it reaches a compiled-in limit,
 which is 300 seconds (5 minutes).
 
+## Serial and parallel image pulls
+
+By default, kubelet pulls images serially. In other words, kubelet will send
+only
+one image pull request to the image service at a time. Other image pull requests
```

Comment:
What do you mean by 'image' service? Are you talking about the component inside kubelet?
Users may get confused because they don't know the internals and they are not interested in them.

```suggestion
one image pull request to the image registry at a time. Other image pull requests
```

Reply:
Image service is the service that handles all image requests from kubelet. Similar to container runtime, it is something configurable on kubelet. See `--image-service-endpoint` on https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/